Administrator Guide/Split brain and ways to deal with it.md

```diff
@@ -0,0 +1,61 @@
+# Split brain and the ways to deal with it
+
+### Split brain:
+Split brain is a situation where two or more replicated copies of a file become divergent. When a file is in split brain, there is an inconsistency in either data or metadata of the file amongst the bricks of a replica and do not have enough information to authoritatively pick a copy as being pristine and heal to the bad copies, despite all bricks being up and online. For a directory, there is also an entry split brain where a file inside it has different gfid/file-type across the bricks of a replica. Split brain can happen mainly because of 2 reasons:
+1. Due to network disconnect:
+Where a client temporarily loses connection to the bricks.
+2. Gluster brick processes going down or returning error:
+Where server1 goes out of service and writes happen on server2. Server1 comes back but server2 goes out of service without the files having been healed. Write occurs on server1 and when server2 is returned to service, each has writes independent of one another.

```

Comment:
While Gluster developers can make sense of this very easily, users may not. It would be best if you re-worded this in a better way.